src/components/Integrations/Shopify/Shopify.js

```diff
@@ -73,7 +75,7 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
                 <div className="block dp-integration__status">
                   <div className="status__info">
                     <div>
-                      {/*<img src="img/integration__logo.png" alt="Shopify logo"/> this should be added as a class, and will be added later*/}
+                      <img src={logo} alt="Shopify logo" />
```

Comment:
What do you think about using class for show logo the Shopify or integrations? As Doppler logo? 
`<h1 class="logo-doppler-new"><a target="_blank" href="https://www.fromdoppler.com/?utm_source=app&amp;utm_medium=landing&amp;utm_campaign=login" rel="noopener noreferrer">Doppler</a></h1>`